nomos-core/risc0_proofs/ptx/Cargo.toml

```diff
@@ -1,18 +1,17 @@
 [package]
-name = "ptx"
-version = "0.1.0"
 edition = "2021"
 license = "MIT or Apache-2.0"
+name    = "ptx"
+version = "0.1.0"
 
 [workspace]
 
 [dependencies]
-risc0-zkvm = { version = "1.0", default-features = false, features = ['std'] }
-serde = { version = "1.0", features = ["derive"] }
-cl = { path = "../../cl" }
+cl                     = { path = "../../cl" }
```

Comment:
Can we use the workspace one?

Reply:
Unfortunately not. These risc0 proofs require their own workspace, and are not part of the top-level workspace (they are also explicitly excluded from it in the top-level `Cargo.toml` file).